spock-core/src/main/java/org/spockframework/compiler/SpecParser.java

```diff
@@ -176,6 +180,31 @@ private void buildFeatureMethod(MethodNode method) {
     spec.getMethods().add(feature);
   }
 
+  private boolean isVerifyMethod(MethodNode method) {
+    boolean hasVerifyAnnotation = !method.getAnnotations(nodeCache.Verify).isEmpty();
+    boolean hasVerifyAllAnnotation = !method.getAnnotations(nodeCache.VerifyAll).isEmpty();
+    if (hasVerifyAnnotation && hasVerifyAllAnnotation) {
+      errorReporter.error(method, "Method '%s' cannot be annotated with both @Verify and @VerifyAll", method.getName());
```

Comment:
The error message below ends with a dot.

```suggestion
      errorReporter.error(method, "Method '%s' cannot be annotated with both @Verify and @VerifyAll.", method.getName());
```